### ISSUE TITLE
Styling is broken on modern info windows

### ISSUE THREAD

**Issue by PerryRylance** [CodeCabin/wp-google-maps#214]
![image](https://cloud.githubusercontent.com/assets/14136738/23555312/5f120846-0020-11e7-9544-c612c5953486.png)

**Comment by PerryRylance** [comment]
This is because of the line

```width: 35% !important;

(selector .wpgmza_pro_modern_circular .wpgmza_modern_infowindow_inner in the windows respective stylesheet)

**Comment by PerryRylance** [comment]
Fixed in #215
